verification/dependencies/net/netip/netip.gobra

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package netip defines an IP address type that's a small value type.
+// Building on that [Addr] type, the package also defines [AddrPort] (an
+// IP address and a port) and [Prefix] (an IP address and a bit length
+// prefix).
+//
+// Compared to the [net.IP] type, [Addr] type takes less memory, is immutable,
+// and is comparable (supports == and being a map key).
+
+// +gobra
+
+package netip
+
+import (
+  	def "github.com/scionproto/scion/verification/utils/definitions"
+)
+
+
+type Addr struct {
+	privateField def.PrivateField
+}
+
+decreases
+func (ip Addr)String() string
```

Comment:
```suggestion
decreases
func (ip Addr) String() string
```

Reply:
Termination is assumed, not proven, so we should use `decreases _`

Reply:
Done.